backend/src/test/java/hu/flowacademy/worksheet/controller/PartnerControllerTest.java

```diff
@@ -59,6 +61,17 @@ public void create() {
         createPartner();
     }
 
+    @Test
+    public void testFilerUserReturnList() {
+        Partner[] toCheckArray = given()
+                .header(getAuthorization(adminLogin()))
+                .param("searchCriteria", "Teszt")
+                .when().get("api/partners")
+                .then()
+                .log().all()
+                .extract().body().as(Partner[].class);
```

Comment:
Ne extractold, ha csak az értéket akarod vizsgálni, arra vannak a assert metódusok.

Reply:
Valami ilyen talán segíthet `.body("$[0].nev", is("Teszt"))`